src/handlers/contrib-reward.ts

```diff
@@ -0,0 +1,61 @@
+import { Context } from "../types";
+
+export async function contribReward(context: Context) {
+  const {
+    logger,
+    payload,
+    octokit,
+    // config: { configurableResponse, },
+  } = context;
+  const contributors: Record<string, number> = {};
+
+  const sender = payload.comment.user?.login;
+  const repo = payload.repository.name;
+  const issueNumber = payload.issue.number;
+  const owner = payload.repository.owner.login;
+  const body = payload.comment.body;
+
+  if (!body.match(/rewards/i)) {
+    logger.error(`Invalid use of slash command, use "/rewards".`, { body });
```

Comment:
I don't understand this entire conditional block's purpose.

Reply:
SInce, I used `hello-world.ts` file as reference, I thought may be this would help in some error handling and just imitated as it is.

Reply:
Removed in the next commit.